BattleshipConsoleGame/BattleshipConsoleGame/Program.cs

```diff
@@ -4,350 +4,371 @@
 
 namespace BattleshipGame
 {
-    class Program
+class Program
+{
+    public static void Main(string[] args)
     {
-        public static void Main(string[] args)
+        Game game = new Game();
+        Printer.DisplayStartMenu();
+        while (!game.GameOver())
         {
-            int gridWidth = 11;
-            int gridHeight = 11;  //one more than standard size of a Battleship playing field to account for labels for the rows and columns on the user board. variables are used for the rest of the program, so if the size needs changed, it can be done here
+            Console.Clear();
+            game.DisplayUserBoard();
+            UserCoordinates userTarget = GetUserTarget();
+            game.ProcessUserTarget(userTarget);
+        }
+        Printer.DisplayWin();
```

Comment:
You want your Main() function to be small and readable--you also want *every* function to be small and readable. So I tried to break up Main() and put it into the Game.ProcessUserTarget() method